test/mplex.spec.js

```diff
@@ -67,4 +68,15 @@ describe('multiplex-generic', () => {
       })
     )
   })
+
+  it('handle aborted stream', (done) => {
+    const aborter = abortable()
+    pull(listenerSocket, aborter)
+
+    aborter.abort()
+    listener.newStream((err) => {
+      expect(err).to.exist()
+      done()
+    })
+  })
```

Comment:
Add tests to check this behavior to https://github.com/libp2p/interface-stream-muxer